modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorBuilder.es.js

```diff
@@ -10,6 +10,10 @@ import {CONJUNCTIONS} from '../../utils/constants.es';
 import Conjunction from './Conjunction.es';
 
 class ContributorBuilder extends React.Component {
+	static propTypes = {
+		onQueryChange: null
```

Comment:
Should this be defaulted to an empty function instead?
```javascript
onQueryChange: () => {}
```

Reply:
it was late 🤦‍♂️